data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1625 @@
+
+
+
+mission "Remnant: Cognizance 36"
+	name "Activate Ssil Vida Power"
+	description "Accept this to signal everyone to restore the power."
+	job
+	repeat
+	source "Ssil Vida"
+	to offer
+		has "Remnant: Cognizance 34: done"
+		not "Remnant: Ssil Vida Active"
+	on accept
+		event "remnant: ssil vida activation"
+		conversation
+			`With a shudder that is felt in every fiber of your being, you glimpse the ripple that makes reality look like the surface of a pond after a single rain drop. Almost before it begins, everything is still again. You look at your sensors, and it looks like Postverta is once again overflowing with life.`
+	on enter Postverta
+		fail
+
+
+
+mission "Remnant: Cognizance 37"
+	name "Take data to Viminal."
+	description "The rest of the Remnant need to know what is happening. Take this data to them."
+	landing
+	source "Ssil Vida"
+	destination "Viminal"
+	to offer
+		has "Remnant: Cognizance 34: done"
+	on offer
+		event "remnant: cognizance salvage expansion"
+	on complete
+		conversation
+			`A crowd of Remnant cluster around your ship as you land, but it is just one who gestures quickly, asking if the rest of the team who went with you are safe. You nod, and a ripple of relief goes through the crowd. "So, what happened?" asks another.`
+			choice
+				`	(Explain everything that happened.)`
+					goto cognizanceexplanation
+				`	"I think I should tell this to the prefects first."`
+			`	The Remnant around you look amused. "Do not be concerned, <first>. Everything you say will reach them. The sooner you tell us, the sooner we can take start preparing for whatever is necessary.`
+			choice
+				`	(Explain everything that happened.)`
+					goto cognizanceexplanation
+				`	"I thought the prefects were your leaders?"`
+			` "They bear our executive responsibilities, it is true. Times of critical threats require the capacity for decisions to be made in the instant. But do not confuse the authority to make high-value decisions with being in charge of everything. The less immediate a decision is, the more likely it is to be the product of significant discussion among us all. Likewise, decisions about what constitute the policy and values of our society are things decided upon by the unified whole, not any one individual." She pauses, and smiles. "Admittedly, given your involvement in smaller projects and unfolding events, our society probably does have a certain autocratic appearance."`
+			choice
+				`	"OK, I will explain everything."`
+					goto cognizanceexplanation
+				`	"How can you reach consensus so easily?"`
+			branch cognizanceuntrusted
+				has "remnant untrusted"
+			`	She looks around with an uncertain expression, then nods. "As I am sure has occurred to you, communication by gestures has some disadvantages. We still prefer it for some tasks, but it can be awkward to watch or gesture in the midst of some situations. While we have some vocabulary we can do without needing hands or arms, it is very limited.`
```

Comment:
```suggestion
			`	She looks around with an uncertain expression, then nods. "As I am sure has occurred to you, communication by gestures has some disadvantages. We still prefer it for some tasks, but it can be awkward to watch or gesture in the midst of some situations. While we have some vocabulary we can use without needing hands or arms, it is very limited.`
```